### ISSUE TITLE
Bug: Setting Initiative when 0 Hp, Giving Dying/Wounded not working

### ISSUE THREAD

**Issue by Fyrexian** [xdy/xdy-pf2e-workbench#346]
Foundry V9.269
System: 3.12.2.10
Only workbench enabled

Not sure why, but now these options don't work for me anymore, and I tested the workbench on a new world, without any other modules enabled. 
I get this error where the key seems to change everytime I change their hp state to 0 or above.

![image](https://user-images.githubusercontent.com/47077661/180311951-77e44afd-ade9-41f2-8354-631e41dd37c9.png)
![image](https://user-images.githubusercontent.com/47077661/180312025-7a8ae73c-f836-43a2-b0ca-37cd37bac1e8.png)
![image](https://user-images.githubusercontent.com/47077661/180312077-9b2cb5ea-d705-4f62-9358-0d7d655188ad.png)

**Comment by xdy** [comment]
Well, I just released a fix. It was *way* too invasive (and really pointed to me having made some bad architecture choices) but seems to work. Modulo what I gave up on. :)
(See the release notes. TL/DR: Players should apply damage and healing to their own actors or things don't work.)
So, Works On My Machine(tm). Prove me wrong! :)
I.e. close this issue if it works for you after updating, or tell me what doesn't work.

**Comment by git-kor-av** [comment]
I was just on my way to open a similar issue.

I have 3.40.3 of workbench with PF2E System 3.13.4.15 & Foundry 9.269 (so latest of everything).

The following settings work fine **as a GM**, but don't seem to work **for Players**:
Automatically increase Dying at 0 HP -> Add 1 + Wounded level, for characters+familiars
Automatically remove Dying when healed to above 0 HP -> Remove Dying for characters+familiars
Automatically give unconscious if Dying removed at 0 HP
Automatically remove unconscious if healed to greater than 0 HP 

I tried disabling any modules that may interact.  When I go from X HP to 0 HP using a player, this is what shows up in console:
```
Workbench increaseDyingOnZeroHP complete [xdy-pf2e-workbench.bundle.js:1:181185](http://192.168.178.43:30000/modules/xdy-pf2e-workbench/xdy-pf2e-workbench.bundle.js)
Workbench autoRemoveUnconsciousAtGreaterThanZeroHP complete [xdy-pf2e-workbench.bundle.js:1:181880](http://192.168.178.43:30000/modules/xdy-pf2e-workbench/xdy-pf2e-workbench.bundle.js)
Foundry VTT | Created ChatMessage with id [4YkfwH01xV31ZeH0] [backend.mjs:306:18](http://192.168.178.43:30000/common/abstract/backend.mjs)
Workbench reminderBreathWeapon complete [xdy-pf2e-workbench.bundle.js:1:175034](http://192.168.178.43:30000/modules/xdy-pf2e-workbench/xdy-pf2e-workbench.bundle.js)
Workbench autoRemoveDyingAtGreaterThanZeroHP complete [xdy-pf2e-workbench.bundle.js:1:181466](http://192.168.178.43:30000/modules/xdy-pf2e-workbench/xdy-pf2e-workbench.bundle.js)
```

**Comment by xdy** [comment]
Just released 3.41.0 which *hopefully* fixes this.

**Comment by xdy** [comment]
Closing as it Works On My Machine(tm). :)
